doc/chapters/service-manifests/index.rst

```diff
@@ -0,0 +1,65 @@
+
+.. _service-manifests:
+
+Service manifests
+*****************
+
+Service manifests are JSON files mapping capabilities to gateway
+configurations. In other words, service manifests describe what
+exact gateway configurations are needed to configure the relevant
+gateways for a specific service in order to realize a capability.
+The service manifests contain the definitions of all the capabilities
+relevant for a service.
+
+For example, a platform might have a capability `com.acme.SomeResource`
+defined. This capability requires access to service `A`. Service `A`
+will then need to deploy a service manifest which contains all gateway
+configurations needed for the `com.acme.SomeResource` capability.
+
+A service manifest might also contain gateway configurations for more
+capabilities.
+
+More services than one might be involved to realize a capability and
+then each relevant service would need to be integrated in this way.
+
+For information about how to work with the manifest files, e.g. where to
+install them etc., see :ref:`Integration <integration>`.
```

Comment:
The "Manifest format" is a sub-heading to this text. My intention is to describe what the reader should expect to find in the rest of the chapter. I will add a note on that this chapter is for the format of the file, and then the reference to how to use it might make more sense.